iOS/Source/MainController.swift

```diff
@@ -1,121 +1,236 @@
-import CoreLocation
 import SweetUIKit
 import UIKit
+import TinyConstraints
 
 class MainController: UIViewController {
-    lazy var informationController: InformationController = {
-        let informationController = InformationController()
-        informationController.modalTransitionStyle = .crossDissolve
-        informationController.delegate = self
+    let insets = UIEdgeInsets(top: 40, left: 40, bottom: 40, right: 40)
 
-        return informationController
+    private let dayLightModelController: DaylightModelController
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .default
+    }
+
+    override var prefersStatusBarHidden: Bool {
+        return true
+    }
+
+    private lazy var informationButton: InformationButton = {
+        let button = InformationButton()
+        button.addTarget(self, action: #selector(didTapAboutButton(button:)), for: .touchUpInside)
+
+        return button
     }()
 
-    lazy var locationTracker: LocationTracker = {
-        let tracker = LocationTracker()
-        tracker.delegate = self
+    private lazy var sunView: SunView = { SunView() }()
 
-        return tracker
+    private lazy var messageLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = Theme.light(size: 32)
+        label.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I think now that you're using `TinyConstraints` you no longer need to add this manually